volatility/framework/plugins/windows/getsids.py

```diff
@@ -0,0 +1,257 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+# Creator: Aviel Zohar (memoryforensicsanalysis@gmail.com) # Fix vol2 plugin to vol3
+import logging
+from typing import Callable, List, Generator, Iterable, Dict
+import re, ntpath
+
+from volatility.framework import renderers, interfaces, objects, exceptions
+from volatility.framework.configuration import requirements
+from volatility.framework.objects import utility
+from volatility.framework.renderers import format_hints
+from volatility.plugins.windows import pslist
+from volatility.framework.layers.registry import RegistryFormatException

```

Comment:
We prefer importing modules rather than classses, in order to prevent people importing getsids to get to `RegistryFormatException` for example